include/enclave/enclave.h

```diff
@@ -9,6 +9,7 @@ class Enclave {
     private:
         // Private constructor to prevent instancing
         Enclave() {}
+        int num_clients;
```

Comment:
I must've missed the addition of this file in a previous PR -- what's the purpose of this file?

Reply:
I think this file has always been around? It essentially maintains a single reference to the enclave object at the host.